src/libraries/System.Text.Json/tests/Serialization/CollectionTests/CollectionTests.Generic.Read.cs

```diff
@@ -257,6 +257,66 @@ public static void ReadSimpleGenericIList()
             Assert.Equal(0, result2.Count());
         }
 
+        [Fact]
+        public static void ReadGenericStructIList()
```

Comment:
It would be good to have one more test (for serialization and deserialization) showing that a custom struct collection can also be an object member e.g.

```cs
public class ClassWithGenericStructIListWrapper
{
    public GenericStructIListWrapper<int> List { get; set; } // Deserializing null should give `JsonException`. Test what happens when serializing `default`.
}
```

```cs
public struct StructWithNullableGenericStructIListWrapper
{
    public GenericStructIListWrapper<int>? List; // Also test when the field is `null` for both serialization and deserialization.
}
```